<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IVTutor</class>
 <widget class="qMRMLWidget" name="IVTutor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>807</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="connectDeviceButton">
     <property name="text">
      <string>Connect Device</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="applyTransformsButton">
     <property name="text">
      <string>Apply Transforms</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="stepsTextDisplay">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>125</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="prevStepButton">
       <property name="text">
        <string>Previous Step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextStepButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Next Step</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Needle Angle:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
